### PR TITLE
Past court reports like present

### LINKED ISSUE #3216: Make Past Court Reports Like Present Court Reports
## What type(s) of user does this feature affect?
 - volunteers
 - supervisors
 - admins
 
## Description
Currently when downloading a court report there are actually two very different documents that might be generated. We only want to keep one of these document formats. Everywhere that there is currently a report being generated with the template: ```default_past_court_date_template.docx``` we want to replace with a court report based on the appropriate default report template (i.e. default in cases that aren't specified and CASA specific templates where which CASA is specified and a specific template exists). 
We want these reports to populate with the appropriate case contacts for the period in question. Essentially, we want court reports that look like someone downloaded a court report the day before the court report due date for any given past court date.  

 court report templates are set at the casa org level

## Screenshots of current behavior, if any
The document template we are trying to eliminate: 
Past Court Date Default Template

## QA Login Details
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by danaguilar**
### What github issue is this PR for, if any?
Resolves #3216

### What changed, and why?
The old court date template was no longer desired. Instead all court date reports now use the default court template

### How will this affect user permissions?
- Volunteer permissions: no affect
- Supervisor permissions: no affect
- Admin permissions: no affect

### How is this tested? (please write tests!) 💖💪
The old tests have been updated to make use of the new reports

### Screenshots please :)
Court date reports looks like this now
new_report_template

### Sample report

Below is a sample report generated by the existing template

[CINA-09-1002 - Court Date - 2023-07-01(3).docx](https://github.com/rubyforgood/casa/files/11994185/CINA-09-1002.-.Court.Date.-.2023-07-01.3.docx)

### CODE REVIEW

File: spec/requests/court_dates_spec.rb [hunk 1]
```diff
@@ -63,63 +64,7 @@
 
         document_inspector = DocxInspector.new(docx_contents: response.body)
 
-        expect(document_inspector.word_list_document_contains?(court_date.date.to_s)).to eq(true)
```
Comment (compwron) [inline spec/requests/court_dates_spec.rb:66]: deleted tests? :(

Comment (danaguilar) [reply spec/requests/court_dates_spec.rb:66]: The new template doesn't have a judge name field or a hearing type field. My understanding was that these fields are no longer wanted and so I removed the tests that checked for them. Would it be better to keep a test the ensured they weren't there anymore?

Comment (compwron) [reply spec/requests/court_dates_spec.rb:66]: Court dates should definitely have judge and hearing type, so something is wrong here...

Comment (danaguilar) [reply spec/requests/court_dates_spec.rb:66]: I must have misunderstood the request then. I thought they wanted to use an existing document. What you're say is that we want to make a new document that's themed like an existing document but with different data. I'll talk with the issue holder to clarify.

**Comment by danaguilar** [comment]
Going to close this PR as I believe I misunderstood what the request was.

**Pull request closed without merging**